src/middlewares/errorHandler.middleware.ts

```diff
@@ -1,10 +1,19 @@
 import { NextFunction, Request, Response } from "express";
 
-export const notFoundHandler = (req: Request, res: Response, next: NextFunction) => {
- res.status(404).json({ message: 'Resource not found' })
-}
+export const notFoundHandler = (
+  _req: Request,
+  res: Response,
+  _next: NextFunction,
+): void => {
+  res.status(404).json({ message: "Resource not found" });
```

Comment:
I propose creating enums HttpStatusCodes and ResponseMessages. Then we would be without hard coding statuses and messages for response.

Reply:
Thanks for the comment, added HttpStatusCodes and ResponseMessages enums